schedulingApp/unit-tests/test_accounts.py

```diff
@@ -51,105 +49,108 @@ def setUp(self):
 
+    def test_blankPermission(self):
         self.assertRaises(TypeError, self.theUser.setPermission(),
                           msg="Calling setPermission with no argument should throw TypeError")
 
-    def test_invalid(self):
-        self.setUp()
 
+    def test_invalidFirstName(self):
         self.assertRaises(ValueError, self.theUser.setUserName(""),
                           msg="Calling setUserName with no argument should throw ValueError")
 
+    def test_invalidLastName(self):
         self.assertRaises(ValueError, self.theUser.setLastName(""),
                           msg="Calling setLastName with no argument should throw ValueError")
 
-        self.assertRaises(ValueError, self.theUser.setPassword(""),
-                          msg="Calling setPassword with no argument should throw ValueError")
-
+    def test_invalidEmailAddress(self):
         self.assertRaises(ValueError, self.theUser.setEmailAddress(""),
                           msg="Calling setEmailAddress with no argument should throw ValueError")
 
+    def test_invalidHomeAddress(self):
         self.assertRaises(ValueError, self.theUser.setHomeAddress(""),
                           msg="Calling setHomeAddress with no argument should throw ValueError")
 
+    def test_invalidPhoneNumber(self):
         self.assertRaises(ValueError, self.theUser.setPhoneNumber(""),
                           msg="Calling setPhoneNumber with no argument should throw ValueError")
 
+    def test_invalidPermission(self):
         self.assertRaises(ValueError, self.theUser.setPermission(""),
                           msg="Calling setPermission with no argument should throw ValueError")
 
-    def test_change(self):
-        self.setUp()
 
+    def test_changeFirstName(self):
         self.theUser.setFirstName("Rob")
         self.assertEqual(self.theUser.getFirstName(), "Rob",
                          msg="Changing the first name to a new one wasn't successful")
 
+    def test_changeLastName(self):
         self.theUser.setLastName("Adams")
         self.assertEqual(self.theUser.getLastName(), "Adams",
                          msg="Changing the last name to a new one wasn't successful")
 
-        self.theUser.setPassword("cats123")
-        self.assertEqual(self.theUser.getPassword(), "cats123",
-                         msg="Changing the password to a new one wasn't successful")
-
+    def test_changeEmailAddress(self):
         self.theUser.setEmailAddress("arob@uwm.edu")
         self.assertEqual(self.theUser.getEmailAddress(), "arob@uwm.edu",
                          msg="Changing the email to a new one wasn't successful")
 
+    def test_changeHomeAddress(self):
         self.theUser.setHomeAddress("Otherplace Rd 234")
         self.assertEqual(self.theUser.getHomeAddress(), "Otherplace Rd 234",
                          msg="Changing the address to a new one wasn't successful")
 
```

Comment:
I would maybe consider storing phone number as an int? Your call though, not a big deal either way.